site2/docs/concepts-messaging.md

```diff
@@ -226,8 +226,8 @@ When a consumer subscribes to a Pulsar topic, by default it subscribes to one sp
 
 When subscribing to multiple topics, the Pulsar client will automatically make a call to the Pulsar API to discover the topics that match the regex pattern/list and then subscribe to all of them. If any of the topics don't currently exist, the consumer will auto-subscribe to them once the topics are created.
 
-> #### No ordering guarantees
-> When a consumer subscribes to multiple topics, all ordering guarantees normally provided by Pulsar on single topics do not hold. If your use case for Pulsar involves any strict ordering requirements, we would strongly recommend against using this feature.
+> #### No cross-topic ordering guarantees
+> Ordering guarantees given for single topics do not hold across multiple topics. So users need to make sure that all messages that need to be delivered in order are on the same topic.
```

Comment:
> #### Message order is not guaranteed for multiple topics
> The order of messages sent by a producer to a single topic is guaranteed. However, if a consumer subscribes to multiple topics, the order of messages is not guaranteed. To guarantee the order of messages to be delivered, you need to make sure that all messages are on the same topic.

Reply:
Imo something like "cross-topic ordering" or "ordering across topics" makes it more clear than "multiple topics" that in-topic ordering is not affected by this.

Reply:
If you think it's more clear to use "cross-topic ordering" or "ordering across topics", it's OK to use that.
I prefer "ordering across topics".

Reply:
Could you please update the content accordingly? Thank you.